### DISCUSSION TITLE
Clicking at 276 BPM after triggering any sound using any power source

### DISCUSSION THREAD

**Discussion by dgtl311** [ElectroTechnique/electrotechnique.github.io#60] [Hardware Issues]
I built my TSynth over a year ago and haven't been able to use it due to a consistent clicking noise coming out of the main audio output. I clocked it at 276 beats per minute. It starts only after triggering a sound via MIDI. The volume pot does not affect the level of the clicking noise. No other pots affect the clicking either. It seems like it is some kind of noise coming from a clock. Once it starts, it doesn't stop until I of course turn it off. I can select any of the patches and the clicking remains constant without skipping a beat. I have tried numerous different power sources and USB cables from three different phone chargers, grounded power strip USB port, different AC power outlets and even an expensive isobar filtered power bank. I tried using several different USB cables including ones that have the ferrite chokes. I already added the resistor mod that is supposed to reduce noise when I built it out as well. I just updated it to the latest firmware hoping that would resolve it but the issue persists. Any advice or suggestions on this would be GREATLY appreciated since I am dying to use my TSynth with my rig!

**Comment by Electrotechnique** [comment]
Can you post a link to a an audio sample of the noise please. I can usually recognise the main noises now. It is possibly the SPI SCK noise still coming through. Can you swap the connection to the headphone with the line out on the audio board and use that?

**Reply by dgtl311** [reply]
[TSynth Click.zip](https://github.com/ElectroTechnique/electrotechnique.github.io/files/6851813/TSynth.Click.zip)

I attached a recording of the clicking that I am hearing. I took the ends and back panel off so I could access the headphone jack. I tried using the headphone jack at full volume with the same cable I used on the line out using an adapter as well as two different audio cables and I still hear the clicking noise. **Although when trying two different sets of headphones on the headphone jack I AM NOT hearing the clicking noise.** I hear the clicking noise on three different sets of powered speakers including professional powered studio monitors. Every component in my signal chain is well grounded and I have no noise issues like this with any of my other sound sources which are using the same audio components. As stated before I have tried powering the TSynth from different isolated, grounded and filtered outlets from the rest of my gear as well as different USB cables and from a computer but the clicking remains. What could be causing this clicking noise? Thank you in advance!

**Reply by Electrotechnique** [reply]
Are you using a mono jack plug? The line out needs stereo the same as the headphone.

**Reply by dgtl311** [reply]
Well I feel like an idiot! That was the problem - it needed a 1/4" **stereo** TRS audio cable. I'm so used to using mono cables for all of my other gear and didn't even think about that! Thank you for your time!
